src/Symfony/Bundle/FrameworkBundle/Command/ConfigDumpReferenceCommand.php

```diff
@@ -112,6 +113,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $this->validateConfiguration($extension, $configuration);
 
         $format = $input->getOption('format');
+
+        if ('yaml' === $format && !class_exists(Yaml::class)) {
+            $errorIo->error('Setting the "format" option to "yaml" requires the "Yaml" component. Either install "symfony/yaml" or use "xml".');
```

Comment:
`Setting the "format" option to "yaml" requires the "Yaml" component. Try running "composer install "symfony/yaml" or use "--format=xml" instead.`